tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/TestContext.cs

```diff
@@ -65,6 +66,8 @@ internal class TestContext
 
         public GetRoadMapBySolutionIdHandler GetRoadMapBySolutionIdHandler => (GetRoadMapBySolutionIdHandler)_scope.GetRoadMapByIdHandler;
 
+        public GetIntegrationsBySolutionIdHandler GetIntegrationsBySolutionIdHandler => (GetIntegrationsBySolutionIdHandler)_scope.GetIntegrationsByIdHandler;
```

Comment:
Should rename to have solutionId beforehand

Reply:
👍